app/src/main/res/layout/overlay_add_on_progress.xml

```diff
@@ -5,6 +5,7 @@
 
 <androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/overlay_card_view"
```

Comment:
Not sure where we need this?

Reply:
Is needed in `AddonsManagementFragment.kt` in order to execute commands that were initially applied to the `include` parent, that now no longer work.
Example:
`addonProgressOverlay?.visibility = View.VISIBLE`
TO
`binding.addonProgressOverlay.overlayCardView.visibility = View.VISIBLE`